@chivay I see there one problem on `ofborg` with `linuxKernel.packages.linux_rt_5_10.lkrg`:
Logs here: https://logs.nix.ci/?key=nixos/nixpkgs.159830&attempt_id=a1a87ac0-e150-4674-853b-63202e3052ab
```
In file included from /build/source/src/modules/ksyms/p_resolve_ksym.c:19:
/build/source/src/modules/ksyms/../../p_lkrg_main.h:438:3: error: #error "LKRG does not support RT kernels (PREEMPT_RT is enabled)"
  438 |  #error "LKRG does not support RT kernels (PREEMPT_RT is enabled)"
      |   ^~~~~
  CC [M]  /build/source/src/modules/database/p_database.o
make[3]: *** [/nix/store/vy02sj1y4lq6k4a9l5pbqbdc7lkbilnd-linux-5.10.152-rt75-dev/lib/modules/5.10.152-rt75/source/scripts/Makefile.build:286: /build/source/src/modules/ksyms/p_resolve_ksym.o] Error 1
make[2]: *** [/nix/store/vy02sj1y4lq6k4a9l5pbqbdc7lkbilnd-linux-5.10.152-rt75-dev/lib/modules/5.10.152-rt75/source/Makefile:1837: /build/source] Error 2
make[1]: *** [/nix/store/vy02sj1y4lq6k4a9l5pbqbdc7lkbilnd-linux-5.10.152-rt75-dev/lib/modules/5.10.152-rt75/source/Makefile:185: __sub-make] Error 2
make[1]: Leaving directory '/nix/store/vy02sj1y4lq6k4a9l5pbqbdc7lkbilnd-linux-5.10.152-rt75-dev/lib/modules/5.10.152-rt75/build'
```

--- a/pkgs/os-specific/linux/lkrg/default.nix
+++ b/pkgs/os-specific/linux/lkrg/default.nix
@@ -1,0 +1,53 @@
+{ lib, stdenv, fetchpatch, fetchFromGitHub, kernel }:
+let
+  isKernelRT = (kernel.structuredExtraConfig ? PREEMPT_RT) && (kernel.structuredExtraConfig.PREEMPT_RT == lib.kernel.yes);
+in
+stdenv.mkDerivation rec {
+  name = "${pname}-${version}-${kernel.version}";
+  pname = "lkrg";
+  version = "0.9.5";
+
+  src = fetchFromGitHub {
+    owner = "lkrg-org";
+    repo = "lkrg";
+    rev = "v${version}";
+    sha256 = "sha256-+yIKkTvfVbLnFBoXSKGebB1A8KqpaRmsLh8SsNuI9Dc=";
+  };
+  patches = [
+    (fetchpatch {
+      name = "fix-aarch64.patch";
+      url = "https://github.com/lkrg-org/lkrg/commit/a4e5c00f13f7081b346bc3736e4c035e3d17d3f7.patch";
+      sha256 = "sha256-DPscqi+DySHwFxGuGe7P2itPkoyb3XGu5Xp2S/ezP4Y=";
+    })
+  ];
+
+  hardeningDisable = [ "pic" ];
+
+  nativeBuildInputs = kernel.moduleBuildDependencies;
+
+  makeFlags = kernel.makeFlags ++ [
+    "KERNEL=${kernel.dev}/lib/modules/${kernel.modDirVersion}/build"
+  ];
+
+  dontConfigure = true;
+
+  prePatch = ''
+    substituteInPlace Makefile --replace "KERNEL := " "KERNEL ?= "
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    install -D lkrg.ko $out/lib/modules/${kernel.modDirVersion}/extra/lkrg.ko
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "LKRG Linux Kernel module";
+    longDescription = "LKRG performs runtime integrity checking of the Linux kernel and detection of security vulnerability exploits against the kernel.";
+    homepage = "https://lkrg.org/";
+    license = licenses.gpl2Only;
+    maintainers = with maintainers; [ chivay ];
+    platforms = platforms.linux;
+    broken = kernel.kernelOlder "5.10" || kernel.kernelAtLeast "6.1" || isKernelRT;
+  };
+}

--- a/pkgs/top-level/linux-kernels.nix
+++ b/pkgs/top-level/linux-kernels.nix
@@ -336,6 +336,8 @@ in {
 
     liquidtux = callPackage ../os-specific/linux/liquidtux {};
 
+    lkrg = callPackage ../os-specific/linux/lkrg {};
+
     v4l2loopback = callPackage ../os-specific/linux/v4l2loopback { };
 
     lttng-modules = callPackage ../os-specific/linux/lttng-modules { };

